streaming-architecture.md

```diff
@@ -1,3 +1,3 @@
 ## Better JSON through streams
```

Comment:
Woops, not sure how I missed `its`! The other one was an error from deleting the bullet point list.